tests/ui/associated-type-bounds/bad-universal-in-impl-sig.stderr

```diff
@@ -0,0 +1,8 @@
+error: associated type bounds are only allowed in where clauses and function signatures, not in impl header
```

Comment:
"impl headers" would probably be better here too, but this one sounds better to me than the one about bounds.